src/helpers/getValue/getAccountDelegationDetails.ts

```diff
@@ -0,0 +1,53 @@
+import BigNumber from 'bignumber.js';
```

Comment:
would be very nice to add some unit tests here, it will help a lot in the future

Reply:
The setup for spec testing for multiple helper functions will come at a later date in a different PR